components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/mediators/XMLSchemaValidator.java

```diff
@@ -0,0 +1,276 @@
+/*
```

Comment:
Add some debug logs so it's easy to diagnose any issue.

Reply:
added debug logs